src/modules/Auth/Components/ChangePassword/ChangePassword.tsx

```diff
@@ -1,9 +1,318 @@
+              type={showOldPassword ? "text" : "password"}
+              variant="outlined"
+              fullWidth
+              margin="normal"
+              {...register("oldPassword", PasswordValidation)}
+              error={!!errors.oldPassword}
+              helperText={errors.oldPassword ? errors.oldPassword.message : ""}
+              InputProps={{
+                endAdornment: (
+                  <InputAdornment position="end">
+                    <IconButton
+                      aria-label={
+                        showOldPassword ? "Hide password" : "Show password"
+                      }
+                      onClick={handleClickShowOldPassword}
+                      onMouseDown={handleMouseDownPassword}
+                      edge="end"
+                    >
+                      {showOldPassword ? <VisibilityOff /> : <Visibility />}
+                    </IconButton>
+                  </InputAdornment>
+                ),
+              }}
+            />
+
+            <Typography
+              variant="subtitle1"
+              sx={{
+                marginBottom: "-0.1rem",
+                color: "var(--primary-color)",
+                fontWeight: "bolder",
+              }}
+            >
+              New Password
+            </Typography>
+            <TextField
+              placeholder="Please type here"
+              variant="outlined"
+              type={showNewPassword ? "text" : "password"}
+              fullWidth
+              {...register("newPassword", PasswordValidation)}
+              error={!!errors.newPassword}
+              helperText={errors.newPassword ? errors.newPassword.message : ""}
+              InputProps={{
+                endAdornment: (
+                  <InputAdornment position="end">
+                    <IconButton
+                      aria-label={
+                        showNewPassword ? "Hide password" : "Show password"
+                      }
+                      onClick={handleClickShowNewPassword}
+                      onMouseDown={handleMouseDownPassword}
+                      edge="end"
+                    >
+                      {showNewPassword ? <VisibilityOff /> : <Visibility />}
+                    </IconButton>
+                  </InputAdornment>
+                ),
+              }}
+            />
+
+            <Typography
+              variant="subtitle1"
+              sx={{
+                marginTop: "5%",
+                marginBottom: "-0.1rem",
+                color: "var(--primary-color)",
+                fontWeight: "bolder",
+              }}
+            >
+              Confirm New Password
+            </Typography>
+            <TextField
+              placeholder="Please type here"
+              variant="outlined"
+              type={showConfirmNewPassword ? "text" : "password"}
+              fullWidth
+              {...register("confirmNewPassword", PasswordValidation)}
```

Comment:
We'll need to make sure that both new and confirm password are matching each other.